src/app/globals.css

```diff
@@ -86,3 +86,25 @@
     @apply bg-background text-foreground;
   }
 }
+input[type="date"],
+input[type="month"] {
+  border: none;
+  box-sizing: border-box;
+  outline: 0;
+  position: relative;
+  width: 100%;
+}
+
+input[type="date"]::-webkit-calendar-picker-indicator,
+input[type="month"]::-webkit-calendar-picker-indicator {
+  background: transparent;
+  bottom: 0;
+  color: transparent;
+  cursor: pointer;
+  height: auto;
+  left: 0;
+  position: absolute;
+  right: 0;
+  top: 0;
+  width: auto;
+}
```

Comment:
Why you have this style instead of tailwind styling ?

Reply:
Actually it is not using now so i removed it now